pkg/cloud/ovirt/providerIDcontroller/providerIDController.go

```diff
@@ -0,0 +1,159 @@
+package providerIDcontroller
+
+import (
+	"context"
+	"fmt"
+
+	ovirtsdk "github.com/ovirt/go-ovirt"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/klog"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+
+	"github.com/openshift/cluster-api-provider-ovirt/pkg/cloud/ovirt/clients"
+)
+
+var _ reconcile.Reconciler = &ProviderIDReconciler{}
+
+type ProviderIDReconciler struct {
+	operatorName         string
+	client               client.Client
+	listNodesByFieldFunc func(key, value string) ([]corev1.Node, error)
+	fetchProviderIDFunc  func(string) (string, error)
+	ovirtApi             *ovirtsdk.Connection
+}
+
+func (p *ProviderIDReconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+	klog.Infof("Reconciling providerID for node %s", request.NamespacedName)
```

Comment:
with the even rate for nodes updates this log is probably gonna be too verbose, may be V(3) would be more appropiate?

Reply:
You might also want to leverage controller runtime built in logger interface e.g https://github.com/openshift/cluster-api-provider-aws/blob/master/cmd/manager/main.go#L140-L144

Reply:
done